src/main/scala/org/openeo/opensearch/OpenSearchResponses.scala

```diff
@@ -251,6 +260,7 @@ object OpenSearchResponses {
             nominalDate <- c.downField("properties").downField("startDate").as[ZonedDateTime]
             links <- c.downField("properties").downField("links").as[Array[Link]]
             resolution = c.downField("properties").downField("resolution").as[Double].toOption
+            publishedDate = c.downField("properties").downField("published").as[ZonedDateTime].toOption
```

Comment:
can we add the parsing of the 'published' property also to the 'parse' implementation of 'FeatureCollection'?
That way we already have it available, there's also a test resource called oscarsProductResponse.json that can be used to validate parsing.

Reply:
Parsed the property everywhere where possible.